### PR TITLE
Added and improved translations in French

### PR DESCRIPTION
**Pull request by Poslovitch**
Also fixed a typo in the English/original translation

#86

There is still some work but I don't think I'll be able to get much more done before a few weeks.

### CODE REVIEW

File: 3denEnhanced/stringtable.xml [hunk 1]
```diff
@@ -2077,7 +2082,7 @@
         <Original>Defines whether the unit can hack hostile UAVs.</Original>
         <English>Defines whether the unit can hack hostile UAVs.</English>
         <Czech>Určuje jestli může jednotka přebrat kontrolu nad nepřátelskými bezpilotními prostředky.</Czech>
-        <French>Défini si l'unité peut ou non pirater un UAV (Drone) hostile.</French>
+        <French>Définti si l'unité peut pirater un UAV (Drone) hostile.</French>
```
Comment (Poslovitch) [inline 3denEnhanced/stringtable.xml:2085]:
```suggestion
        <French>Définit si l'unité peut pirater un UAV (Drone) hostile.</French>
```

File: 3denEnhanced/stringtable.xml [hunk 2]
```diff
@@ -2011,7 +2016,7 @@
         <Original>Defines whether the unit can fully heal other units.</Original>
         <English>Defines whether the unit can fully heal other units.</English>
         <Czech>Určuje jestli může jednotka plně léčit ostatní jednotky.</Czech>
-        <French>Défini si l'unité peut soigner complètement d'autres unités.</French>
+        <French>Définti si l'unité peut soigner complètement d'autres unités.</French>
```
Comment (Poslovitch) [inline 3denEnhanced/stringtable.xml:2019]:
```suggestion
        <French>Définit si l'unité peut soigner complètement d'autres unités.</French>
```

**Comment by Poslovitch** [comment]
Since I'm doing a lot of small commits (I'm working on both my laptop and desktop, when I have some free time), I recommend you to squash & merge, instead of just merging.

**Comment by R3voA3** [comment]
Alright, let me know when you are done with all the tweaks so I can squash and merge it.
Thanks for the contribution!

**Comment by Poslovitch** [comment]
That's it!

**Comment by Poslovitch** [comment]
@R3voA3 In case you forgot about this PR - it's completed ;)

**Pull request merged**